pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -34,7 +37,13 @@ def hoc_s(id, markdown: false, locals: nil)
     end
   end
 
-  string = @actionview.render(inline: string, type: :safe_md) if markdown
+  if markdown
+    type = markdown == :inline ? :inline_md : :safe_md
+    if type == :inline_md
+      puts "rendering #{string.inspect} inline"
```

Comment:
Do we want to print a message each time we encounter one of these?

Reply:
oops! Stray debug statement